bin/qc-headers

```diff
@@ -271,14 +316,20 @@ def main(dicom, standard, output):
 
     mismatches = compare_headers(standardHeader, dicomHeader, tolerances=tolerances, ignore=ignore)
 
+    if dti:
+        dti_diffs = check_bvecs(dicom, standard)
+        mismatches.extend(dti_diffs)
+
     print('{} mismatches for {}'.format(len(mismatches), dicom))
+
+    print(mismatches)
```

Comment:
This print statement should maybe be removed, was it just there for debugging?

Reply:
It should be removed! I'll fix